src/renderer/webgl/shaders/RadialGradientShader.mjs

```diff
@@ -21,115 +21,128 @@ import DefaultShader from "./DefaultShader.mjs";
 import StageUtils from "../../../tree/StageUtils.mjs";
 
 export default class RadialGradientShader extends DefaultShader {
-    
     constructor(context) {
         super(context);
-        
-        this._x = 0;
-        this._y = 0;
-
-        this.color = 0xFFFF0000;
-
-        this._radiusX = 100;
-        this._radiusY = 100;
+        this._anchor = [0, 0];
+        this._ic = 0xffffffff;
+        this._normalizedIC = this._getNormalizedColor(this._ic);
+        this._oc = 0x00ffffff;
+        this._normalizedOC = this._getNormalizedColor(this._oc);
+        this._radius = 0;
     }
 
-    set x(v) {
-        this._x = v;
+    set radius(v) {
+        this._radius = v;
         this.redraw();
     }
 
-    set y(v) {
-        this._y = v;
+    set innerColor(argb) {
+        this._ic = argb;
+        this._normalizedIC = this._getNormalizedColor(argb);
         this.redraw();
     }
 
-    set radiusX(v) {
```

Comment:
Fixed, the radiusY is now supported again. Do however notice that it will be slightly different, this is because the old shader never projected the correct diameters. Somewhere the assumption was made that diameters === radius. Which is not.

Reply:
I did some tests and it looks to me the original `RadialGradient` shader is completely broken so I don't think it's used anywhere (although might be wrong), so even though technically it breaks compatibility it actually fixes the shader so we'll keep the change, worst case scenario we'll revert it.